region/brinstar/red/Beta Power Bomb Room.json

```
@@ -272,10 +272,24 @@
       "link": [1, 3],
       "name": "Morph Power Beam Sidehopper Kill",
       "requires": [
-        "Morph"
+        "Morph",
```

```suggestion
        "Morph",
        "canCameraManip",
```
This is chaos if fighting more than one at a time.